Marlin/language_an.h

```diff
@@ -148,8 +172,19 @@
 #define MSG_BABYSTEP_Y                      "Babystep Y"
 #define MSG_BABYSTEP_Z                      "Babystep Z"
 #define MSG_ENDSTOP_ABORT                   "Endstop abort"
+#define MSG_HEATING_FAILED_LCD              "Heating failed"
```

Comment:
When a language is missing a translation for one directive it's better **not to include** the English directive on the language file as we now have a script which searches for [missing translations](https://github.com/MarlinFirmware/Marlin/blob/RCBugFix/Marlin/scripts/findMissingTranslations.sh) and outputs them.

The script is unable to detect when a language file has the **directive defined in English** thus makes the language files hard to maintain.